src/providers/MeatInfoProvider.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, createContext, useEffect } from 'react'
+
+
+export const MeatInfoContext = createContext();
+
+
+export const MeatInfoProvider = (props) => {
+    const [selectedMeatItemInfoFromSearchBar, setSelectedMeatItemInfoFromSeachBar] = useState('');
```

Comment:
This name is way too long (like many others)
Please try to think of how you can describe them in a short and clear way, and which words are simply not necessry.